requirements.txt

```diff
@@ -5,4 +5,6 @@ base58
 multidict<5.0.0
 yarl==1.6.0
 
-indy_vdr
\ No newline at end of file
+# indy_vdr
+# Error loading library: indy_vdr specified version because it occurs when installing as pip3 install indy-vdr.
+indy_vdr==0.4.0.dev1
```

Comment:
Please update to “0.4.0.dev16” and confirm it is still working.  Sorry I missed that dev1 was not the latest.

Thanks

Reply:
After modifying to dev16, it was confirmed that it also works normally when running.
And then I commit again